docs/workloads/azure/backend/java_cqrs/testing_java_cqrs.md

```diff
@@ -70,7 +70,7 @@ and the equivalent for running the Integration tests.
 #### Test reports
```

Comment:
The same changes that are to be made in the above file should be made in this one